openstack/networking/v2/extensions/trunks/results.go

```diff
@@ -0,0 +1,101 @@
+package trunks
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+type Subport struct {
+	SegmentationID   int    `json:"segmentation_id"`
+	SegmentationType string `json:"segmentation_type"`
+	PortID           string `json:"port_id"`
+}
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// CreateResult is the response from a Create operation. Call its Extract method
+// to interpret it as a Trunk.
+type CreateResult struct {
+	commonResult
+}
+
+type Trunk struct {
```

Comment:
Just for posterity / reference:

https://github.com/openstack/neutron/blob/master/neutron/objects/trunk.py#L91-L99

https://github.com/openstack/neutron/blob/master/neutron/objects/trunk.py#L131

https://github.com/openstack/neutron/blob/master/neutron/objects/trunk.py#L142